ocrd_zip.md

```diff
@@ -127,25 +130,26 @@ be referenced in a `mets:file/mets:Flocat` in the `mets.xml`.
 To pack a workspace to OCRD-ZIP:
```

Comment:
If only absolute paths are allowed and all files in the data directory must be referenced in METS, the workspace actually consists only of the METS file.

Reply:
That was the original idea. But it's a hassle in reality, hence the workspace concept which combines METS with downloaded files.